README.md

```diff
@@ -1,313 +1,386 @@
 # sveltekit-view-transition
 
-An abstraction on top of `view-transition-api` and `onNavigate` to allow for a seamless experience in SvelteKit.
+Simplified `view-transition-api` for Sveltekit.
 
-[![MIT License](https://img.shields.io/badge/License-MIT-green.svg)](https://choosealicense.com/licenses/mit/)
+[![MIT License](https://img.shields.io/badge/License-MIT-green.svg)](https://choosealicense.com/licenses/mit/) &nbsp; ![GitHub last commit](https://img.shields.io/github/last-commit/paoloricciuti/sveltekit-view-transition) &nbsp; ![npm](https://img.shields.io/npm/v/sveltekit-view-transition) &nbsp; ![npm](https://img.shields.io/npm/dt/sveltekit-view-transition) &nbsp; ![npm bundle size](https://img.shields.io/bundlephobia/minzip/sveltekit-view-transition)
 
-![npm bundle size](https://img.shields.io/bundlephobia/minzip/sveltekit-view-transition)
+<br>
```

Comment:
I don't think you need `<br />` in markdown files, you just need an extra `\n`

Reply:
Ah yea I like to use a br for more visual separation between major sections because their renderer ignores extra newlines